src/main/resources/config.yml

```diff
@@ -21,6 +21,7 @@ enable-whois-on-player-join: true
 enable-whois-on-command: true
 whois-admin-permission-name: ww.admin
 whois-no-match-found-message: "&4&lNo matching player found with your whois query"
+whois-player-on-first-join-fireworks: false
```

Comment:
Plz rename this variable to "firework-on-join". and add it to the last of the config.yml

Reply:
okay give me a minute